### PR TITLE
{data}[intel/2018.00] netCDF v4.5.0

### PR DESCRIPTION
**Pull request by reedts**
(created using `eb --new-pr`)

edit: depends on #5310

### CODE REVIEW

File: easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2018.00'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = [
+    'https://github.com/Unidata/netcdf-c/archive/'
+]
```
Comment (boegel) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb]: @reedts Please include a SHA256 checksum (`eb --inject-checksums` can help), and collapse `source_urls` to a single line

File: easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+dependencies = [
+    ('HDF5', '1.10.1'),
+    ('cURL', '7.55.1'),
+    ('Szip', '2.1.1'),
+]
+
+builddependencies = [
+    ('Autotools', '20170619'),
```
Comment (boegel) [inline easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb]: Hmm, this is new, why did you add this specifically?

We don't have this in existing `netCDF` easyconfig, if you added this for good reason, we should do this in other `netCDF` easyconfigs too.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
node2612.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/84689344528179a62afbd9cfe1236c34 for a full test report.

File: easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb (hunk 2 above)
Comment (reedts) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb]: @boegel The `netCDF-intel-2017a` easyconfig ([here](https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-intel-2017a.eb)) includes this as builddependency. That's why I included it too (in newer version).

Comment (boegel) [reply easybuild/easyconfigs/n/netCDF/netCDF-4.5.0-intel-2018.00.eb]: This was done in #5077, but apparently it's not fully consistent yet... Thanks for the reply.

In any case, this looks good to go.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cfde5fb2aeba6817623b233c99419f44 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2461.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4d41f04b6bb06ba75129cc99244a92e7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @reedts!
